openqabot/amqp.py

```diff
@@ -105,3 +139,51 @@ def handle_incident(self, inc_nr: int, message: Dict[str, Any]) -> None:
 
     def handle_aggregate(self, unused_build: str, unused_message: Dict[str, Any]) -> None:  # noqa: ARG002 Unused method argument
         return
+
+    def handle_inc_review_request(self, message: Dict[str, Any], args: Namespace) -> None:  # noqa: ARG002 Unused method argument
+        pr_number = int(message.get("pull_request", {}).get("number"))
+        if not pr_number:
+            log.error("Could not extract PR number from AMQP message")
+            return
```

Comment:
I think `int` will throw a `ValueError` but never `0` (or another falsy value) if an argument is given. So this error handling is dead code.